plugin/kubernetes/handler_test.go

```diff
@@ -288,6 +288,7 @@ var svcIndex = map[string][]*api.Service{
 				Protocol: "tcp",
 				Port:     80,
 			}},
+			Type: api.ServiceTypeExternalName,
```

Comment:
Agree.. view from e2e tests of k8s. The externalNames services are defined by type.